### PR TITLE
Create PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by CaroleAnneDaunais**
This template will allow consistency for code reviewing.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Description
```
Comment (blaisb) [inline .github/PULL_REQUEST_TEMPLATE.md:1]:
```suggestion
# Description of the problem
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- Explain the previous issue (description of the issue or reference to the number of an issue)
```
Comment (blaisb) [inline .github/PULL_REQUEST_TEMPLATE.md:3]:
```suggestion
- Explain the previous issue (description of the issue or reference to the number of an issue)
- Explain the need for a new feature or component
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+- Describe the added code
+
+# How Has This Been Tested?
+
+- List and describe the test(s) for the added code (if there is no test for the pull request, explain why (i.e. the test already exists but was changed))
+
```
Comment (blaisb) [inline .github/PULL_REQUEST_TEMPLATE.md:9]:
```suggestion
# Description of the solution

- Explain the solution to the existing issue
- Explain the content of the new feature

# How Has This Been Tested?

- List and describe the test(s) for the added code (if there is no test for the pull request, explain why (i.e. the test already exists but was changed))

```

**Review by blaisb — APPROVED** [review]
I think breaking down the first category into two is a better idea.
First we state the problem, then we state the solution.
It will be a bit clearer.

otherwise, Amazing job.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# Description of the problem
+
+- Explain the previous issue (description of the issue or reference to the number of an issue)
```
Comment (Luckabarbeau) [inline .github/PULL_REQUEST_TEMPLATE.md:3]:
```suggestion
- Explain the previous issue (description of the issue or the reference number of an open issue)
```

**Review by Luckabarbeau — APPROVED** [review]
Small recommendation for the text. Everything seems good to me otherwise

**Pull request merged**